src/Connection.php

```diff
@@ -67,6 +101,30 @@ public function getChannel()
         return $this->channel;
     }
 
+    /**
+     * Fetch the list of consumers details for the given queue name using the management API.
+     *
+     * @param  string $queueName
+     * @param  string $columns
+     *
+     * @return array
+     */
+    public function fetchQueueConsumers($queueName, string $columns = 'consumer_details.consumer_tag')
```

Comment:
@harris21 @KinaneD not sure if this should happen here, it was done here to not share the credentials elsewhere. There could've been a `Management` entity for the request details. Thoughts?

Reply:
What about the rest of the comment? how would auth happen without sharing credentials?

Reply:
Will leave this as an enhancement, could be to provide accessors to credentials and separate the client as you suggested.

Reply:
Let's keep it here for now, I cannot thing of any other place for the credentials at the moment. 💃